Where2Play/Models/MusicBrainz.cs

```diff
@@ -6,110 +6,109 @@
 //
 //    var welcome = Welcome.FromJson(jsonString);
 
-namespace Where2Play.Models
-{
-    using System;
-    using System.Collections.Generic;
+namespace Where2Play.Models;
 
-    using System.Globalization;
-    using Newtonsoft.Json;
-    using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
 
-    public partial class MusicBrainzArtist
-    {
-        [JsonProperty("gender-id")]
-        public object GenderId { get; set; }
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 
-        [JsonProperty("name")]
-        public string Name { get; set; }
+public partial class MusicBrainzArtist
+{
+    [JsonProperty("gender-id")]
+    public object GenderId { get; set; }
 
-        [JsonProperty("begin-area")]
-        public Area BeginArea { get; set; }
+    [JsonProperty("name")]
+    public string Name { get; set; }
```

Comment:
It looks like the order was changed a bit here, as the added lines don't necessarily match up with the removed lines... but if I look around the file, I can find where the added and removed lines line up.
Is there a good reason to change the order?  If not, I'd leave it as-is.